src/App.js

```diff
@@ -4,9 +4,32 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    movies: moviesFromServer,
+    searchInput: '',
+  };
+
+  searchHandler = event => (
+    this.setState({
+      searchInput: event.target.value,
+    }, this.filetVisibleMovies)
+  )
+
+  filetVisibleMovies = () => {
```

Comment:
filet?
![filet](https://user-images.githubusercontent.com/66570142/105478013-4c0cc380-5cab-11eb-89b9-131388478a19.png)

Reply:
Ou... It  should have been - filter )))